### ISSUE TITLE
Bug in align.py (perform_align) causes the software to fail to align to absolute frame

### ISSUE THREAD

**Issue by stscijgbot-hstdp** [spacetelescope/drizzlepac#1237]
_Issue [HLA-689](https://jira.stsci.edu/browse/HLA-689) was created on JIRA by [Michele De La Pena](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mdelapena):_

This is Git Issue #&#x2060;1236.  I manually have duped it here.

A bug in perform_align() causes the processing to return from the current routine which indicates the dataset could not align to an absolute astrometric frame. A "KeyErrror: 1" is raised. Error happens
```java
            # Get the location of the current image in the filtered table
            index = np.where(alignment_table.filtered_table['imageName'] == imgname)[0][0]

            # First ensure sources were found

            if table is None or not table[1]:    *** Error happens here. ***
                log.warning("No sources found in image {}".format(imgname))
                alignment_table.filtered_table[:]['status'] = 1
                alignment_table.filtered_table[:]['processMsg'] = "No sources found"
                log.info(make_label('Processing time of [STEP 4]', starting_dt))
                alignment_table.close()
                return None code placeholder
```
As a test I substituted the error line with just "if table is None", and the data was able to align to an absolute frame.

**Comment by stsci-hack** [comment]
Closed by #1250.

**Comment by stscijgbot-hstdp** [comment]
_Comment by [Michele De La Pena](https://jira.stsci.edu/secure/ViewProfile.jspa?name=mdelapena) on [JIRA](https://jira.stsci.edu/browse/HLA-689?focusedCommentId=566776#comment-566776):_

Sorry, the dataset which triggered this issue was ieps22040.
